doc/examples/simulate_dki.py

```diff
@@ -0,0 +1,83 @@
+"""
+==========================
+DKI MultiTensor Simulation
+==========================
+
+In this example we show how someone can simulate the diffusion tensor,
```

Comment:
I'm not a native English speaker, but can't really hold a conversation in
my mother tongue anymore either.

I think "we show how to simulate the diffusion tensor ..." would work well
here.

On Mon, Apr 27, 2015 at 4:42 PM, Samuel St-Jean <notifications@github.com>
wrote:

> In doc/examples/simulate_dki.py
> <https://github.com/nipy/dipy/pull/582#discussion_r29203137>:
>
> > @@ -0,0 +1,83 @@
> > +"""
> > +==========================
> > +DKI MultiTensor Simulation
> > +==========================
> > +
> > +In this example we show how someone can simulate the diffusion tensor,
>
> I think it's more common to say we show one can instead of someone. Is
> there actually a native english speaker we can poke for that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/582/files#r29203137>.
>